portal-web/test/functional/com/liferay/portalweb/paths/analyticscloud/ACSegments.path

```diff
@@ -63,6 +63,11 @@
 	<td>xpath=(//div[contains(@class,'criterion')][.//div[contains(@class,'display-value')][normalize-space(.)='${criteriaName}']]//button[contains(@class,'dropdown-toggle')])[${index}]</td>
 	<td></td>
 </tr>
+<tr>
+	<td>CRITERIA_ELLIPSIS_OPTION_OPEN</td>
+	<td>xpath=(//div[contains(@class,'criterion')][.//div[contains(@class,'display-value')][normalize-space(.)='birthDate']]//button[contains(@class,'dropdown-toggle') and @aria-expanded='true'])</td>
```

Comment:
Hey @joshitagaki, I think you forgot to change the name of the criteria to use a variable instead of always using the `birthDate`
```suggestion
	<td>xpath=(//div[contains(@class,'criterion')][.//div[contains(@class,'display-value')][normalize-space(.)='${criteriaName}']]//button[contains(@class,'dropdown-toggle') and @aria-expanded='true'])</td>
```

Reply:
Oops good catch!